src/config.js

```diff
@@ -3,9 +3,10 @@ import NamedError from './NamedError'
 let s3 = new AWS.S3()
 
 export function fetchConfig () {
+  console.log('Fetching configuration file from S3.')
   return new Promise((resolve, reject) => {
-    let stage = process.env.Stage
-    if (stage !== 'CODE' && stage !== 'PROD') {
+    let stage: ?string = process.env.Stage
+    if (stage == null || !['CODE', 'PROD'].includes(stage)) {
       reject(new Error(`invalid stage: ${stage}, please fix Stage env variable`))
       return
     }
```

Comment:
just a comment for later: maybe we could put this code somewhere shared as most lambdas need to know the stage